mne/forward/_lead_dots.py

```diff
@@ -162,7 +187,43 @@ def _comp_sums_meg(beta, ctheta, lut_fun, n_fact, volume_integral):
 
 def _fast_sphere_dot_r0(r, rr1, rr2, lr1, lr2, cosmags1, cosmags2,
                         w1, w2, volume_integral, lut, n_fact, ch_type):
-    """Lead field dot product computation for M/EEG in the sphere model"""
+    """Lead field dot product computation for M/EEG in the sphere model.
+
+    Parameters
+    ----------
+    r : float
+        The integration radius. It is used to calculate beta as:
+        beta = (r * r) / (lr1 * lr2).
+    rr1 : array, shape (n_points x 3)
+        Normalized position vectors of integrations points in first sensor.
+    rr2 : array, shape (n_points x 3)
+        Normalized position vector of integration points in second sensor.
+    lr1 : array, shape (n_points x 1)
+        Magnitude of position vector of integration points in first sensor.
+    lr2 : array, shape (n_points x 1)
+        Magnitude of position vector of integration points in second sensor.
```

Comment:
We're inconsistent here. Lately we were more strict about it, adopting sklearn conventions. If you're lazy keep it for a big PR once some one is bored.